addons/hr_holidays/models/hr_leave_type.py

```diff
@@ -247,9 +259,10 @@ def name_get(self):
             if record.allocation_type != 'no':
                 name = "%(name)s (%(count)s)" % {
                     'name': name,
-                    'count': _('%g remaining out of %g') % (
+                    'count': _('%g remaining out of %g %s') % (
```

Comment:
Avoid breaking the translations, keep the string as it is and append hours or days.